src/main/resources/config/application.yml

```diff
@@ -276,6 +276,9 @@ info:
     display-ribbon-on-profiles: "dev"
     sentry:
         dsn: https://8c6b41ec2d4245e8bd3ec9541d53f625@sentry.io/1440029 # Leave empty to disable Sentry, must be a valid URI
+    postHog: # Leave empty to disable PostHog
+        host: https://analytics.artemis.cit.tum.de
+        token: phc_GNN4LDYNMi5eJcwpFGla9miaOihku7jCsL9erEWs8ak
```

Comment:
not sure if we should expose the token publicly here. Could this be a security risk?